resources/application-connector/charts/connector-service/templates/helm-certs-removal-backup.yaml

```diff
@@ -0,0 +1,98 @@
+{{- range $key, $value := .Values.certs -}}
```

Comment:
What is the goal of this code? Maybe I'm missing something but checking disableLegacyConnectivity flag is enough.

Reply:
The aim of this fragment is to apply migration logic to both of 
`certs:
  cert1:
    name: connector-service-app-ca
    namespace: kyma-integration
  cert2:
    name: app-connector-certs
    namespace: istio-system
`
certificates that previously were regenerated in cert setup job if at leas one of them was empty.

Reply:
So, we have four jobs here. Two for backup, two for restore. Theoretically if we would need to handle both secrets we could doi it in one job. But this is not a main concern here.
Maybe I don't  why we even touch `app-connector-certs`? This one is already deleted from chart.
The backup and restore jobs are needed for `connector-service-app-ca` since we removed `connector-service-app-ca-secret.yaml`  file.

Reply:
You are right. Previously I have made a foolish mistake of running restore after the cert setup That way the cert setup was regenerating both secrets. Now that you have pointed out the correct order I have removed a backup of `app-connector-certs`.